op-conductor/conductor/service_test.go

```diff
@@ -129,7 +129,8 @@ func (s *OpConductorTestSuite) SetupTest() {
 	s.conductor.leaderUpdateCh = s.leaderUpdateCh
 
 	s.err = errors.New("error")
-	s.syncEnabled = false // default to no sync, turn it on by calling s.enableSynchronization()
+	s.syncEnabled = false   // default to no sync, turn it on by calling s.enableSynchronization()
+	s.wg = sync.WaitGroup{} // create new wg for every test in case last test didn't finish the action loop during shutdown.
```

Comment:
With the change below (added `s.wg.Wait()`, this shouldn't happen), but it's good hygiene to re-create it for every test since it is needed for synchronization